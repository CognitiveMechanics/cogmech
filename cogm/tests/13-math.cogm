
// This file tests mathj.cogm from the standard library

@include "../lib/math.cogm"

: Add(1, 2)
: Add(10, 2)
: Add(12643, 75)

: Multiply(2, 3)
: Multiply(10, 5)