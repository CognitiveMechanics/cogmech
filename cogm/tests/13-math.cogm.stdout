3
12
12718
6
50
